scene/2d/navigation2d.cpp

```diff
@@ -671,7 +671,7 @@ Object *Navigation2D::get_closest_point_owner(const Vector2 &p_point) {
 
 				if (Geometry::is_point_in_triangle(p_point, _get_vertex(p.edges[0].point), _get_vertex(p.edges[i - 1].point), _get_vertex(p.edges[i].point))) {
 
-					E->get().owner;
+					return E->get().owner;
```

Comment:
Judging by the equivalent `Navigation` method implemented together with this one in 7648088fca, I guess it might have been meant to be `owner = E->get().owner;`, and the second for loop might be extraneous. But indeed would need to be checked by @reduz if he remembers what he meant to do back then.

Reply:
I think this is right, since if the point is in a triangle, the owner is that triangle

Reply:
guess method was never used much